packages/danger/src/PullRequestManager.ts

```diff
@@ -0,0 +1,174 @@
+import { Octokit } from '@octokit/rest';
+import { GitHubPRDSL } from 'danger/distribution/dsl/GitHubDSL';
+
+import { GithubApiWrapper } from './GithubApiWrapper';
+import { getPackageChangelogRelativePath } from './Utils';
+
+export enum ChangelogEntryType {
+  BUG_FIXES = 0,
+  NEW_FEATURES = 1,
+  BREAKING_CHANGES = 2,
+}
+
+export const DEFAULT_ENTRY_TYPE = ChangelogEntryType.BUG_FIXES;
+
+export type ChangelogEntry = {
+  type: ChangelogEntryType;
+  message: string;
+};
+
+export const DEFAULT_CHANGELOG_ENTRY_KEY = 'default' as const;
+
+export type ChangelogEntries = {
+  [DEFAULT_CHANGELOG_ENTRY_KEY]: ChangelogEntry;
+  [key: string]: ChangelogEntry;
+};
+
+export type PullRequest = GitHubPRDSL | Octokit.PullsListResponseItem;
+
+const dangerMessage = `Add missing changelog`;
+const dangerTags = `[danger][bot]`;
+
+export class PullRequestManager {
+  constructor(private pullRequest: PullRequest, private githubApi: GithubApiWrapper) {}
+
+  /**
+   * Gets suggested changelog entries from PR provided in the constructor.
+   *
+   * If PR doesn't contain `# Changelog` section, this method returns:
+   * {
+   *   [DEFAULT_CHANGELOG_ENTRY_KEY]: <title of this pr without tags>
+   * }
+   * Otherwise, it tries to parse PR's body.
+   */
+  parseChangelogSuggestionFromDescription(): ChangelogEntries {
+    const changelogEntries = {
+      [DEFAULT_CHANGELOG_ENTRY_KEY]: {
+        type: DEFAULT_ENTRY_TYPE,
+        message: this.pullRequest.title.replace(/\[.*\]/, '').trim(),
+      },
+    };
+
+    const parseLine: (line: string) => void = line => {
+      const tags = this.parseTagsFromLine(line);
+      if (!tags) {
+        warn(`Couldn't parse line: ${line}.`);
```

Comment:
Is it a warning that's gonna be a github comment? If so, I think we should be more clear here - for example, for pull request title it should say sth like that: ``I couldn't find any tags in pull request title `<pull request title>`, please follow the instructions in <link to the section in contributing guide that describes how to correctly tag pull requests>``.
For changelog entries it should be something similar, with a small difference because it's not PR's title 😉 
For entries under `# Changelog`, if